src/module/item/spell/document.ts

```diff
@@ -725,7 +725,7 @@ class SpellPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends Ite
                 const overlayIds = [...variant.appliedOverlays!.values()];
                 const actions = (() => {
                     const actionIcon = getActionIcon(variant.system.time.value, null);
-                    return variant.system.time.value !== this.system.time.value && actionIcon ? actionIcon : null;
+                    return actionIcon;
```

Comment:
You can remove the IIFE now. :)

Reply:
Updated. I don't know how I didn't notice that. 😄